src/interfaces/rest.js

```diff
@@ -55,7 +55,21 @@ module.exports = function(app) {
           path[path.length - 1] === 'meta' &&
           path[0] === 'vessels'
         ) {
-          const meta = getMetadata(path.slice(0, path.length - 1).join('.'))
+          const metaPath = path.slice(0, path.length - 1).join('.')
+          let meta = getMetadata(metaPath)
+
+          if (path[1] === app.selfId) {
```

Comment:
Should we support metas from defaults.json outside `vessels` and for `vessels.*`? I mean just replace `selfId` with `'self'`and otherwise use the path as it is?